### PR TITLE
update tostadas container to include additional py libraries required…

### PR DESCRIPTION
**Pull request by jessicarowell**
… for seqsender submission code

The Dockerfile updates TOSTADAS version from 0.2.0-beta to 3.1.0.  The new version contains an updated environment.yml file which adds 3 libraries required by updated SeqSender code:
- lxml 4.9.1
- xmltodict 0.13.0
- nameparser 1.0.5

Pull Request (PR) checklist:
- [x] Include a description of what is in this pull request in this message.
- [x] The dockerfile successfully builds to a test target for the user creating the PR. (i.e. `docker build --tag samtools:1.15test --target test docker-builds/samtools/1.15` )
- [x] Directory structure as name of the tool in lower case with special characters removed with a subdirectory of the version number (i.e. `spades/3.12.0/Dockerfile`)
   - [] (optional) All test files are located in same directory as the Dockerfile (i.e. `shigatyper/2.0.1/test.sh`)
- [x] Create a simple container-specific [README.md](https://github.com/StaPH-B/docker-builds/blob/master/.github/workflow-templates/readme-template.md) in the same directory as the Dockerfile (i.e. `spades/3.12.0/README.md`)
   - [ ] If this README is longer than 30 lines, there is an explanation as to why more detail was needed
- [x] Dockerfile includes the recommended [LABELS](https://github.com/StaPH-B/docker-builds/blob/master/dockerfile-template/Dockerfile#L8-L18) 
- [x] Main [README.md](https://github.com/StaPH-B/docker-builds/blob/master/README.md) has been updated to include the tool and/or version of the dockerfile(s) in this PR
- [x] [Program_Licenses.md](https://github.com/StaPH-B/docker-builds/blob/master/Program_Licenses.md) contains the tool(s) used in this PR and has been updated for any missing

### CODE REVIEW

**Comment by jessicarowell** [comment]
Ok I've changed the base image to micromamba (thanks for your help!) and tested it on our pipeline on-prem. I think it's ready!

File: tostadas/3.1.0/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,59 @@
+FROM mambaorg/micromamba:1.5.8 as app
+
+ARG TOSTADAS_VER="3.0.0"
```
Comment (erinyoung) [inline tostadas/3.1.0/Dockerfile:3]: Should this be 3.1.0?

**Comment by erinyoung** [comment]
Could you adjust the main readme (https://github.com/jessicarowell/docker-builds/blob/master/README.md) to include the new version of this image? Tostadas is line 279 (ish depending on what PRs get merged)

File: tostadas/3.1.0/Dockerfile (hunk 1 above)
Comment (jessicarowell) [reply tostadas/3.1.0/Dockerfile]: yes...fixed! thanks

**Comment by jessicarowell** [comment]
> Could you adjust the main readme (https://github.com/jessicarowell/docker-builds/blob/master/README.md) to include the new version of this image? Tostadas is line 279 (ish depending on what PRs get merged)

done!

**Review by erinyoung — APPROVED** [review]
Looks good!

I added back in the prior tostadas container version to the readme so there is 0.2.0-beta in addition to 3.1.0.

This is going to trigger the tests again. Once those pass, I'll merge this PR and get this deployed.

**Comment by erinyoung** [comment]
Thank you for putting this together!

You can check the status of the deploy here : https://github.com/StaPH-B/docker-builds/actions/runs/8543209592. This image should be available in a few minutes.

**Pull request merged**